app/main.py

```diff
@@ -2,0 +1,6 @@
+def copy_file(command: str) -> None:
+    text = command.split(" ")
```

Comment:
The variable name 'text' is not very descriptive. Consider using a more descriptive name like 'command_parts' to improve code readability.